pl_examples/bug_report_model.py

```diff
@@ -67,7 +77,7 @@ def loss(self, batch, prediction):
         # An arbitrary loss to have a loss that updates the model weights during `Trainer.fit` calls
         return torch.nn.functional.mse_loss(prediction, torch.ones_like(prediction))
 
-    def step(self, x):
+    def _step(self, x):
```

Comment:
This has to be called somewhere. So the reference there Must ne changed as well I think :)

Reply:
I did not find it anywhere, but I'll revert it...